views/api.html

```diff
@@ -0,0 +1,198 @@
+<div class="o-layout o-layout--docs" data-o-component="o-layout">
+	<div class="o-layout__header">
+		{{> header }}
+	</div>
+
+	<div class="o-layout__sidebar"></div>
+
+	<div class="o-layout__main o-layout-typography" data-o-component="o-syntax-highlight">
+		<h1 id="#api-reference">API Reference</h1>
+		<div>
+			<h2 id="get-v3-bundle-js)">JavaScript</h2>
+			<table class="o-table o-table--horizontal-lines o-table--row-headings">
+				<tbody>
+					<tr>
+						<th scope="row">Method</th>
+						<td>
+							<code>GET</code>
+						</td>
+					</tr>
+					<tr>
+						<th scope="row">
+							Path
+						</th>
+						<td>
+							<code>/v3/bundles/js</code>
+						</td>
+					</tr>
+					<tr>
+						<th scope="row">Querystring</th>
+						<td>
+							<dl>
+								<dt>
+									<code>
+										<var>modules</var>
```

Comment:
Feature(kinda) request: Shall we change this to components?

Reply:
I've filled an issue for this work #33